server/src/db.ts

```diff
@@ -86,7 +87,7 @@ export async function createLocation(
   data: any,
   { source = "unknown" } = {}
 ): Promise<ProviderLocation> {
-  data = validateLocationInput(data, true);
+  data = callWithSpan(validateLocationInput, data, true);
```

Comment:
Not sure whether I should have just used `wrapWithSpan` on all the validation function instead, so this *just happens* instead of having to explicitly use `callWithSpan` here.